apimatic_core/types/union_types/union_type_context.py

```diff
@@ -0,0 +1,94 @@
+
+class UnionTypeContext:
+
+    @classmethod
+    def create(cls, is_array=False, is_dict=False, is_array_of_dict=False, is_optional=False, is_nullable=False,
```

Comment:
This is not a constructor but a factory method for context initialization with optional parameters. If we create another data class then this will add another layer to the context initialization flow by codegen and I am afraid that will end up as an over-engineered design.

Reply:
ok